lib/src/util/useEmptyComponent.ts

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect, useCallback, RefObject } from 'react';
+import useMutationObserver from '@rooks/use-mutation-observer';
+
+const mutationObserverConfig = { subtree: false, childList: true, attributes: false, characterData: false };
+
+/**
+ * Convenience hook that shows if a component is empty or not.
+ * This custom hook implements the useMutationObserver hook to check if there is a change in
+ * the parent component's children.
+ * @return a boolean if the component is empty.
+ * Note: elements within the parent component that are mapped as emptyChildRef are
+ * effectively ignored.
+ */
+const useEmptyComponent = (
+  initialValue: boolean,
```

Comment:
Why is this a parameter? Doesn't its value basically not matter since `setIsEmpty` gets called right after mount? I'd just hardcode the `useState` here to start with `false`

Reply:
> Why is this a parameter? Doesn't its value basically not matter since `setIsEmpty` gets called right after mount? I'd just hardcode the `useState` here to start with `false`

Removed the `false` parameter and hardcoded `isEmpty` to be false in `useState`.
29dc2caf958bd9983327ef1f982de59f97e9b475